hnn_core/cell_response.py

```diff
@@ -322,7 +322,8 @@ def mean_rates(self, tstart, tstop, gid_ranges, mean_type='all'):
 
         return spike_rates
 
-    def plot_spikes_raster(self, trial_idx=None, ax=None, show=True):
+    def plot_spikes_raster(self, trial_idx=None, ax=None, start_from_t0=True,
```

Comment:
I don't think you need a new parameter for this ... if users want, they can just do `ax.set_xlim` to change the limits.

Reply:
Another option would be to use ticks instead of dots for the raster plot so that they are more visible if/when the x-axis is cut off at 0. I actually prefer ticks generally speaking.